.github/workflows/build-mudlet.yml

```diff
@@ -266,13 +266,17 @@ jobs:
 
     - name: (Linux) Set build info
       if: runner.os == 'Linux'
+      env:
+        GITHUB_EVENT_INPUTS_SCHEDULED: ${{inputs.scheduled}}
```

Comment:
How does this work and given that you've done this for Linux and MacOs does something need to be done for the Window(s) builds? Though it does seem possible that there is something a bit different in `build-mudlet-win.yml` to do the same thing... {Setting an environmental variable: `GITHUB_SCHEDULED_BUILD` to `${{ github.event_name == 'schedule' || github.event.inputs.scheduled == 'true' }}`) - though I need to read up more to find out which of those is the REAL `cron` schedule build and which is a fake one from somewhere...}

Reply:
Windows builds already work OK for this.